src/app/app-modules/core/header/header.component.ts

```
@@ -131,25 +145,45 @@ export class HeaderComponent implements OnInit, DoCheck, AfterViewInit, OnChange
+    const userRolesString = this.sessionstorage.getItem('userRoles');
     if (userRolesString !== null) {
       this.roles = JSON.parse(userRolesString);
     }
    
   }
 
+  refreshLogin(){
+    this.auth.getUserDetails().subscribe((res: any) => {
+      // console.log(">>>>>>", res);
+      if (res.statusCode === '200') {
+        if (res.data.previlegeObj && res.data.previlegeObj[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)